Resources/Prototypes/Maps/hive.yml

```
@@ -2,8 +2,8 @@
   id: TheHive
   mapName: 'The Hive'
   mapPath: /Maps/hive.yml
-  minPlayers: 20
-  #maxPlayers: 80 Temporary maybe
+  minPlayers: 15
+  maxPlayers: 75 
```

yamllint (1.29.0-1)

[error] trailing spaces

(trailing-spaces)